src/resources/partners.md

```diff
@@ -85,3 +85,45 @@ partners_list:
                 class: youtube
                 url: 'https://www.youtube.com/moduscreate'
 ---
+        name: Monterail
```

Comment:
The addition needs to be moved up before the `---` line to be included in the metadata.